Build fails on CentOS 7

Building the emulator fails on CentOS 7 due to undeclared variables in the for loops.  For example, running make results in several errors such as this.

```c
c -O3 -Wall -Werror -g -I/usr/include/SDL2 -D_REENTRANT -c video.c -o video.o
In file included from video.c:11:0:
gif.h: In function ‘GifPartition’:
gif.h:167:3: error: ‘for’ loop initial declarations are only allowed in C99 mode
   for (int ii = left; ii < right - 1; ++ii) {
   ^
gif.h:167:3: note: use option -std=c99 or -std=gnu99 to compile your code
```
After adding -std=c99 to the CFLAGS I then get another error.

```
09:59 $ make
cc -std=c99 -O3 -Wall -Werror -g -I/usr/include/SDL2 -D_REENTRANT -c main.c -o main.o
main.c: In function ‘main’:
main.c:230:21: error: ‘PATH_MAX’ undeclared (first use in this function)
  char rom_path_data[PATH_MAX];
                     ^
main.c:230:21: note: each undeclared identifier is reported only once for each function it appears in
main.c:586:5: error: implicit declaration of function ‘usleep’ [-Werror=implicit-function-declaration]
     usleep(1000 * diff_time);
     ^
main.c:230:7: error: unused variable ‘rom_path_data’ [-Werror=unused-variable]
  char rom_path_data[PATH_MAX];
       ^
cc1: all warnings being treated as errors
make: *** [main.o] Error 1
```

How can I resolve this?  I was able to build the ROM without an issue however I'm not sure where the emulator expects the file to be located.

--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@ else
 	SDL2CONFIG=sdl2-config
 endif
 
-CFLAGS=-O3 -Wall -Werror -g $(shell $(SDL2CONFIG) --cflags)
+CFLAGS=-std=c99 -O3 -Wall -Werror -g $(shell $(SDL2CONFIG) --cflags)
 LDFLAGS=$(shell $(SDL2CONFIG) --libs) -lm
 
 ifeq ($(MAC_STATIC),1)

--- a/README.md
+++ b/README.md
@@ -6,7 +6,11 @@ This is an emulator for the Commander X16 computer system. It only depends on SD
 
 Binary releases for macOS, Windows and x86_64 Linux are available on the [releases page](https://github.com/commanderx16/x16-emulator/releases).
 
-For all other systems, make sure the development version of SDL2 is installed and type `make` to build the source. You will also need the system ROM (`rom.bin`), which you can build from the [X16 ROM source](https://github.com/commanderx16/x16-rom) or take from the *latest* binary release. (It is not always guaranteed though that the latest binary release is compatible with the current state of the emulator source.)
+To build the emulator you will need the SDL2 development package along with the [cc65](https://cc65.github.io/) assembler.  On RedHat based nodes these are provided by the SDL2-devel and cc65 packages (yum install SDL2-devel cc65).
+
+You will also need the system ROM (`rom.bin`) which you can build from the [X16 ROM source](https://github.com/commanderx16/x16-rom) or take from the *latest* binary release. (It is not always guaranteed though that the latest binary release is compatible with the current state of the emulator source.)
+
+Type `make` to build the source.
 
 ## Starting
 

--- a/main.c
+++ b/main.c
@@ -2,6 +2,8 @@
 // Copyright (c) 2019 Michael Steil
 // All rights reserved. License: 2-clause BSD
 
+#define _XOPEN_SOURCE   600
+#define _POSIX_C_SOURCE 1
 #include <stdlib.h>
 #include <stdbool.h>
 #include <stdint.h>

